utility/doc/trustless_relay_validation.md

```diff
@@ -0,0 +1,65 @@
+# Validation of Trustless Relays
+
```

Comment:
Table of contents (its usually autogenerated by vscode)

Reply:
Thank you for the review. Fixed.